src/pretix/settings.py

```diff
@@ -845,4 +845,10 @@ def traces_sampler(sampling_context):
 DATA_UPLOAD_MAX_NUMBER_FIELDS = 25000
 DATA_UPLOAD_MAX_MEMORY_SIZE = 10 * 1024 * 1024  # 10 MB
 
+# File sizes are in MB
```

Comment:
Thecomment says MB, the new docs say MiB. I always get confused with the two, which is correct here and should we use it consistently? ;)

Reply:
I guess it is MiB due to the 1024*1024 instead of 1000*1000

Reply:
```suggestion
# File sizes are in MiB
```

Other occurrences of MiB in this file are also wrong.
But I think as MB and MiB are not that far apart, it is not such a big problem.